integration-cli/docker_cli_help_test.go

```diff
@@ -51,19 +52,14 @@ func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
 		helpCmd := exec.Command(dockerBinary, "help")
 		helpCmd.Env = newEnvs
 		out, ec, err := runCommandWithOutput(helpCmd)
-		if err != nil || ec != 0 {
```

Comment:
@runcom It could. But the problem is sometimes it is `cmd.Stdout` or other things, we want to configure soo :sweat:. We probably should have a `Cmd(mycmd)` that would allow to customize these things..

I'm thinking/working a bit on this but.. that's for for right now so I guess for now it's good as is :stuck_out_tongue_closed_eyes: